ex9/js/main.js

```diff
@@ -0,0 +1,38 @@
+requirejs(['ajaxGet','load','filter', 'sort'] , function(ajaxGet, load, filter, sort) {
+  $(function() {
+    var phoneBookData = {};
+    var render = function(objList) {
```

Comment:
Also, please put some comments as to what objList is. Similar to phoneBookData, it is an array of employees (technically a subset of phoneBookData).